snap/snapcraft.yaml

```diff
@@ -22,10 +22,13 @@ parts:
   node-cert-exporter:
     plugin: go
     source: https://github.com/amimof/node-cert-exporter.git
-    source-tag: $SNAPCRAFT_PROJECT_VERSION
+    source-tag: v1.1.4
     source-type: git
     build-snaps:
       - go
+    override-pull: |
+      craftctl default
+      craftctl set version="$(git describe --tags --abbrev=10)"
```

Comment:
We most likely do not want the `v` prefix in the upstream tag to be present in the snap version presented to the end user, would there be any way to filter that out from the version string?

Reply:
Yes, I've included a fix for this.